src/scenes/signin/index.js

```diff
@@ -2,10 +2,17 @@ import React, { Component } from 'react';
 import { Grid, Form, Segment, Menu, Button } from 'semantic-ui-react';
 
 export default class Signin extends Component {
-  state={ selected: 'login' };
+  state = { login: true };
+
+  selectLogin() {
```

Comment:
use arrow functions

`selectLogin = () => { }`

And then remove `.bind(this)` from the function calls.